octobot_trading/data/order.py

```diff
@@ -254,38 +264,67 @@ def is_self_managed(self):
 
     def update_from_raw(self, raw_order):
         if self.side is None or self.order_type is None:
-            self.__update_type_from_raw(raw_order)
-            if self.taker_or_maker is None:
-                self.__update_taker_maker_from_raw()
+            try:
+                self.__update_type_from_raw(raw_order)
+                if self.taker_or_maker is None:
+                    self.__update_taker_maker_from_raw()
+            except KeyError:
+                pass
```

Comment:
Is it really good to silently pass on an order parsing error ?

Reply:
I agree :+1:

Reply:
Can you add an error log or a raise ? 😄

Reply:
Yes sorry ^^